src/xkbcomp/symbols.c

```diff
@@ -1523,7 +1545,8 @@ CopyModMapDefToKeymap(struct xkb_keymap *keymap, SymbolsInfo *info,
     else {
         key = FindKeyForSymbol(keymap, entry->u.keySym);
         if (!key) {
-            log_vrb(info->ctx, 5,
+            log_vrb_with_id(info->ctx, 5,
```

Comment:
I think there's an argument to have `log_vrb()` and friends always take an id and have an `XKB_NO_ERROR_ID = 0` that can be special-cased in the log function implementations. I think there are only a few cases where we don't log with id, so better to optimise for the majority?

Reply:
I agree, I just did not want to start such refactor for now, because we need to agree on previous points and it is going to create a huge diff. What do you prefer:

- Create a further MR to do this refactor.
- Make this refactor in a separate commit: it facilitate reviewing by commit, then it can be squashed.

Reply:
honestly, whatever is the combination of easiest to do vs. easiest to review. I'm happy with this being a follow-up - in the end the effort typing and testing is going to be worse than the effort looking at the patch after so it's up to you